theme/src/components/heading.js

```diff
@@ -9,6 +9,15 @@ import styled from 'styled-components'
 const StyledHeading = styled(Heading)`
   margin-top: ${themeGet('space.4')};
   margin-bottom: ${themeGet('space.3')};
+  
+  // Makes sure links to Headings are not covered by the fixed "main" header
+  // header (66px) + margin (24px) = 90px
```

Comment:
This `90px` magic number will be a bit prone for regressions. Depends on the `margin-top` of the Heading, but also the height of [components/header.js](https://github.com/primer/doctocat/blob/e145052d0a3174ffb11f1685429024bc86a13975/theme/src/components/header.js). Not sure if it's worth trying to calculate it automatically?